src/m_config.c

```diff
@@ -2308,7 +2317,7 @@ static char *GetDefaultConfigDir(void)
     char *result;
     char *copy;
 
-    result = SDL_GetPrefPath("", PACKAGE_TARNAME);
```

Comment:
Change to use wrapper in one commit, then add functionality in another

Reply:
This one is wrong, it will change the previous behavior which used executable directory as fallpath. Working on a better workaround.